### ISSUE TITLE
Transit kms and times in CBA calculation

### ISSUE THREAD

**Issue by atte-wsp** [HSLdevcom/helmet-model-system#178]
Transit vehicle kms and times are different in Helmet 3.1 and 4.0 CBA calculations even if using exact same network files. Difference in times seems natural, but transit kms should be same. 

I'm not sure which method is correct. Test cases show that Helmet 3.1 prints more transit vehicle kilometres and therefore greater impacts than Helmet 4.0. 

This is minor issue, but good to check at some point.

**Comment by zptro** [comment]
This is probably because of different volume factors. What would be the correct volume factors for transit vehicle kms?

**Comment by atte-wsp** [comment]
That's probably true. Currently Helmet 4.0 uses transit demand coefficients `aht *  1. / 0.478 + pt * 1. / 0.109 + iht * 1. / 0.405` and Helmet 3.1 was using `aht * 3 + pt * 10 + iht* 3`.

It think volume factors for vehicle kms should be counted from transit supply during peak and day hours. Earlier I counted number of departures during peak and day hours from HSL GTFS dump (param.volume_factors: "bus"), which could represent overall transit supply sufficiently. 

Additionally I noticed that we could have "bus: @bus" defined in param.link_volumes, so that  24h volumes would be counted for background bus traffic as well.
